ingestion/src/metadata/ingestion/lineage/sql_lineage.py

```diff
@@ -146,9 +146,13 @@ def get_table_entities_from_query(
     split_table = table_name.split(".")
     empty_list: List[Any] = [None]  # Otherwise, there's a typing error in the concat
 
-    database_query, schema_query, table = (
-        empty_list * (3 - len(split_table))
-    ) + split_table
+    if len(split_table) > 3:
```

Comment:
what's the idea behind this?

Reply:
so with bigquery sometimes the table names are like this `<project-id>.<dataset-id>.infromation_schema.<table-name>` we also do the same query [here](https://github.com/open-metadata/OpenMetadata/blob/7524b673d0c4338dca49d6ebfabd9b989bf117d9/ingestion/src/metadata/ingestion/source/database/bigquery/queries.py#L48) so to avoid a value error (too many values to unpack). in case when we have more than 3 values to unpack then we just say that the last one is the table name and move ahead

Reply:
ah I see.. Does this only happen with `INFORMATION_SCHEMA`? or is this something more generic where we can identify db/schema/table info?

Could we please add a comment explaining the scenario and maybe a unit test?

Reply:
It only happens with the `information_schema`, otherwise the dataset would contain the table directly. 

and sure I can add some comments.